src/styles/container.scss

```diff
@@ -0,0 +1,33 @@
+.container {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  grid-template-rows: repeat(2, 1fr);
+  justify-content: center;
+  align-content: center;
+
+  &__cell {
+    display: flex;
+    min-width: $size-cell;
+    height: $size-cell;
+    align-items: center;
+    justify-content: center;
+  }
+
+  @media (max-width: 750px) {
```

Comment:
You should fix max-width in media quaries, because your blocks look no good when window narrowing.
![image](https://user-images.githubusercontent.com/29862525/134138617-ef350a33-98d6-4570-99ad-512759a644d7.png)

Reply:
Solved.